### PR TITLE
Member MVC Created

### PR DESCRIPTION
**Pull request by pburslemjr**
Added member controller. Controller has following actions implemented: index, show, edit.
Index: displays list of members with name, email, service hours, pp %, and missed mandatory events
Show: shows above information for selected member
Edit: Toggle member/admin of account

Also updated events table to only display "attend/unattend" button for passed events.

coding standard; Used resourceful routes. No additional testing

Desc: As an admin, I want to be able to view and edit the profile of any member, so that I can check the participation status of any member as needed

acceptance criteria; Admin should be able to click "details" on any member and be taken to a page that shows participation percentage and total approved and unapproved service hours

Definition of Done: MVC created for User that allows view edit and delete. Resourceful routes are added to relevent views/controllers to allow admins to access the view, and promote users to admin.

### CODE REVIEW

File: app/views/events/_table.html.erb [hunk 1]
```diff
@@ -28,13 +28,16 @@
 
       <td>
 
+      <% if DateTime.now() >= event.datetime %>
```
Comment (asiddiqi18) [inline app/views/events/_table.html.erb:31]: event.datetime might be null

File: app/views/events/_table.html.erb [hunk 2]
```diff
@@ -28,13 +28,16 @@
...
       <% if !current_member.isAdmin %>
+     
         <% if attended %>
```
Comment (asiddiqi18) [inline app/views/events/_table.html.erb:34]: add tabs for better formatting

File: app/views/member/_table.html.erb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<% members.each do |member| %>
+<tr>
+  <td><%= member.name %></td>
+  <td><%= member.email %></td>
+
+  <td>
+  <% if @past_events.length() == 0 %>
+        No events have occured!
+        <% else %>
+        <%= ((member.events.length().to_f / @past_events.length()) * 100.to_f).round(2) %>%
```
Comment (asiddiqi18) [inline app/views/member/_table.html.erb:23]: preferable to move business logic out of HTML

**Review by asiddiqi18 — CHANGES_REQUESTED** [review]
Looks good!
One major issue is that events page returns an error if an event doesn't have a datetime specified to it. Fixing this will require a nil check on an event's datetime.

File: app/views/events/_table.html.erb (hunk 1 above)
Comment (pburslemjr) [reply app/views/events/_table.html.erb]: Fixed if statement to allow user to attend events with datetime=nil

**Comment by pburslemjr** [comment]
Changed code to handle events with nil datetime. Added sorting functionality, but front-end design could use some work still.

File: app/views/member/_table.html.erb (hunk 3 above)
Comment (pburslemjr) [reply app/views/member/_table.html.erb]: Completed this by providing functions within member.rb

**Review by asiddiqi18 — APPROVED** [review]

**Pull request merged**